forge-gui/res/cardsfolder/upcoming/displaced_dinosaurs.txt

```diff
@@ -0,0 +1,8 @@
+Name:Displaced Dinosaurs
+ManaCost:5 G G
+Types:Creature Dinosaur
+PT:7/7
+R:Event$ Moved | ValidCard$ Permanent.Historic+YouCtrl | ActiveZones$ Battlefield | Destination$ Battlefield | ReplaceWith$ BecomeDino | ReplacementResult$ Updated | Description$ As a historic permanent enters the battlefield under your control, it becomes a 7/7 Dinosaur creature in addition to its other types. (Artifacts, legendaries, and Sagas are historic.)
+SVar:BecomeDino:DB$ Animate | Defined$ ReplacedCard | Toughness$ 7 | Power$ 7 | Types$ Dinosaur | Duration$ Permanent
```

Comment:
you should test tricky cards if they work

Reply:
Yeah I should have been more clear with my previous comment that I didn't think it was working ...

Any ideas how to make this work ? Return to the previous iteration ?

Reply:
does it help if you add Creature type back?

Reply:
Thanks working now 
![image](https://github.com/Card-Forge/forge/assets/67333662/e3f5751e-f80d-4d6e-a9ad-cef8303ec12c)